files/lib/data/conversation/ConversationAction.class.php

```diff
@@ -878,6 +896,7 @@ public function validateAddParticipants()
      */
     public function addParticipants()
     {
+        // TODO migrate to FormBuilder
```

Comment:
Outstanding todo?

Reply:
When migrating `WCF.Conversation.InlineEditor` to TypeScript, the FormBuilder should be used